web/src/pages/user.jsx

```diff
@@ -84,7 +119,15 @@ export default ({songs}) => {
                         }}
                     />
                 </div>
-                {searchedSongs.length ? (
+                {loading ? (
+                    <div className="w-full h-full flex flex-col m-auto place-content-center">
+                        <div>
+                            <h3 className="text-accent-neutral text-lg mx-auto text-center">
+                                Loading more songs...
```

Comment:
Would be nice to have some sort of spinning loading indicator rather than just this text... from a user experience perspective helps them to realize the site isn't frozen and that it's actually doing something

Reply:
Spinning indicator has been added while request is loading